recipes/opencl-headers/all/conanfile.py

```diff
@@ -36,7 +36,7 @@ def package(self):
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "OpenCLHeaders")
         self.cpp_info.set_property("cmake_target_name", "OpenCL::Headers")
-        self.cpp_info.set_property("pkg_config_name", "OpenCL")
+        self.cpp_info.set_property("pkg_config_name", "OpenCL-Headers")
```

Comment:
Ok, but PR title is a little bit confusing. This PR fixes pkgconfig name and add new versions.

Reply:
updated title.